src/test/components/ArticleCard.test.ts

```
@@ -0,0 +1,144 @@
+    expect(typeof mockProps.author).toBe("string");
+    expect(typeof mockProps.slug).toBe("string");
+    expect(typeof mockProps.readingTime).toBe("number");
+  });
+
+  it("devrait tronquer la description si elle dépasse 120 caractères", () => {
+    const longDescription = "Cette description est très longue et devrait être tronquée automatiquement pour s'adapter au format compact des cartes d'articles dans la grille principale de la section.";
+    
+    // Reproduire la logique de troncature du composant
+    const truncatedDescription = longDescription.length > 120 
+      ? longDescription.substring(0, 120) + "..." 
+      : longDescription;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)